packages/rest-generator/openapitools.json

```diff
@@ -2,6 +2,6 @@
   "$schema": "node_modules/@openapitools/openapi-generator-cli/config.schema.json",
   "spaces": 2,
   "generator-cli": {
-    "version": "4.3.1"
+    "version": "5.0.0-beta2"
```

Comment:
Question: the `^2.1.4` is used in the package.json, which is a stable version instead of beta. Did you manually modify this line or maybe 2.1.4 points to the beta version?

Reply:
2.1.4 is the version of the CLI, but with the CLI you can configure the version of the generator (through this openapitools.json).